gradle.properties

```diff
@@ -48,20 +48,20 @@ spark_40_antlr_version=4.13.1
 spark_33_jackson_version=2.13.4
 spark_34_jackson_version=2.14.2
 spark_35_jackson_version=2.15.2
-spark_40_jackson_version=2.17.0
+spark_40_jackson_version=2.18.2
 
 spark_33_slf4j_version=1.7.32
 spark_34_slf4j_version=2.0.6
 spark_35_slf4j_version=2.0.7
-spark_40_slf4j_version=2.0.7
+spark_40_slf4j_version=2.0.16
 
 # Align with Apache Spark, and don't bundle them in release jar.
 commons_lang3_version=3.12.0
 commons_codec_version=1.17.2
 
 # javax annotations removed in jdk 11
 # fix build error with jakarta annotations
-jakarta_annotation_api_version=1.3.5
+jakarta_annotation_api_version=2.1.1
```

Comment:
it used to address gRPC-generated code issues, and is useless now, let's remove this dependency directly

Reply:
see https://github.com/ClickHouse/spark-clickhouse-connector/issues/64